### DISCUSSION TITLE
Remove 'substraitVeloxFunctionMap_'

### DISCUSSION THREAD

**Discussion by zhli1142015** [apache/gluten#7920] [Ideas]
Looks some function are reigistered with different names in Gluten and Velox, so we added the map `substraitVeloxFunctionMap_`. Now we alredy have separate spark function registers. Should we rename these fucntions in Velox to remove this map?

```
std::unordered_map<std::string, std::string> SubstraitParser::substraitVeloxFunctionMap_ = {
    {"is_not_null", "isnotnull"}, /*Spark functions.*/
    {"is_null", "isnull"},
    {"equal", "equalto"},
    {"equal_null_safe", "equalnullsafe"},
    {"lt", "lessthan"},
    {"lte", "lessthanorequal"},
    {"gt", "greaterthan"},
    {"gte", "greaterthanorequal"},
    {"not_equal", "notequalto"},
    {"char_length", "length"},
    {"strpos", "instr"},
    {"ends_with", "endswith"},
    {"starts_with", "startswith"},
    {"named_struct", "row_constructor"},
    {"bit_or", "bitwise_or_agg"},
    {"bit_and", "bitwise_and_agg"},
    {"murmur3hash", "hash_with_seed"},
    {"xxhash64", "xxhash64_with_seed"},
    {"modulus", "remainder"},
    {"date_format", "format_datetime"},
    {"negative", "unaryminus"},
    {"get_array_item", "get"}};
```

https://github.com/facebookincubator/velox/pull/11454#discussion_r1837405274

**Comment by zhli1142015** [comment]
cc @jinchengchenghh , @FelixYBW , @zhouyuan , @rui-mo , thanks

**Comment by FelixYBW** [comment]
Yes, Let's remove it.

**Comment by jinchengchenghh** [comment]
I assume not, the function name in substrait origins from https://github.com/substrait-io/substrait/blob/main/extensions/functions_comparison.yaml, and the function name in velox is same with Spark, so we should have a mapping for them.
